### DISCUSSION TITLE
indenation vs code

### DISCUSSION THREAD

**Discussion by broyerdamien** [microsoft/vscode-discussions#423] [Extension Development QnA] [answered]
Hello, I am a beginner in web development, I am on linux. My automatic indenation and some of my extensions no longer work. I uninstalled and reinstalled vscode but still the same problem. Do you have a solution ?

**Comment by gjsjohnmurray** [comment, marked as answer]
This forum focuses on developers who are building extensions for VS Code.

In future, to report a problem as a user of VS Code please start at https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions.

To investigate your current problem I suggest you use Command Palette (F1) to run the Extension Bisect procedure. If this doesn't identify an extension causing the problem, use the link above. Follow the advice, including first searching for similar issues on the main VS Code repository.
